doc/releases/squid.rst

```diff
@@ -0,0 +1,335 @@
+=====
+Squid
+=====
+
+Squid is the 19th stable release of Ceph.
+
+v19.2.0 Squid
+=============
+
+Ceph
+~~~~
+
+* ceph: a new `--daemon-output-file` switch is available for `ceph tell`
+  commands to dump output to a file local to the daemon. For commands which
+  produce large amounts of output, this avoids a potential spike in memory
+  usage on the daemon, allows for faster streaming writes to a file local to
+  the daemon, and reduces time holding any locks required to execute the
+  command. For analysis, it is necessary to retrieve the file from the host
+  running the daemon manually. Currently, only ``--format=json|json-pretty``
```

Comment:
'manually retrieve', instead of the manually at the end (which confusingly refers to the 'running')

Reply:
Fixed